src/doc/rustdoc/src/doc-rendering-tweaks.md

```diff
@@ -0,0 +1,20 @@
+# Documentation rendering tweaks
+
+This chapter is about how to improve the generated documentation appearance.
```

Comment:
I would prefer to put this in the 'advanced features' section - I don't expect many people to use it, and having a whole chapter for the one feature seems like overkill, I think people will expect 'how do I write rust documentation' which is not what this is.

Reply:
We have already a lot of things in there, making access to information more complicated. I expect for us to have more "front-end" tweaks like that so I think it's better to have it in a section on its own.

Reply:
Hmm, if you plan to have more tweaks, does it make sense to plan them out ahead of time? I don't want to land these piece-meal and realize we ended with a bunch of similar ones that don't address people's needs.